rotkehlchen/api/v1/resources.py

```diff
@@ -2858,3 +2847,30 @@ class EnsAvatarsResource(BaseMethodView):
     @use_kwargs(get_schema, location='view_args')
     def get(self, ens_name: str) -> Response:
         return self.rest_api.get_ens_avatar(ens_name=ens_name, match_header=get_match_header())
+
+
+class ClearCacheResource(BaseMethodView):
+    post_schema = ClearCacheSchema()
+    icons_schema = ClearIconsCacheSchema()
+    avatars_schema = ClearAvatarsCacheSchema()
+    price_schema = NamedOracleCacheSchema()
```

Comment:
Undo this entire change please. It's not a good place while keeping the previous endpoint for price oracle caches for everything else